target/linux/lantiq/xrx200/base-files/etc/hotplug.d/firmware/12-ath9k-eeprom

```diff
@@ -19,6 +19,9 @@ case "$FIRMWARE" in
 			avm,fritz7360sl)
 				caldata_extract "urlader" 0x985 0x1000
 				;;
+			avm,fritz3390)
```

Comment:
Please apply alphabetic sorting.

Reply:
I'm unsure how to sort this. The case for 3370 also contains 7362sl. Would you sort 3390 as the second case after 3370|7362sl, but before 7360sl?

Reply:
Yes, the first member determines the sorting of the whole block.